gemonlinedb/src/common/GEMOnlineDBManagerWeb.cc

```diff
@@ -0,0 +1,74 @@
+// GEMOnlineDBManagerWeb.cc
+
+#include "gem/onlinedb/GEMOnlineDBManagerWeb.h"
+
+#include <memory>
+
+#include "xcept/tools.h"
+
+#include "gem/onlinedb/GEMOnlineDBManager.h"
+#include "gem/onlinedb/exception/Exception.h"
+
+gem::onlinedb::GEMOnlineDBManagerWeb::GEMOnlineDBManagerWeb(gem::onlinedb::GEMOnlineDBManager* glibApp) :
+  gem::base::GEMWebApplication(glibApp)
```

Comment:
Copy-paste leftover?

Reply:
Indeed, this PR was just a placeholder to have a MWE, and I just cloned the structure from the GLIB applications :-)
I'll be basically offline today and tomorrow, so if this is a sticking point (or there are other must-be-fixed items), they'll have to wait until Wednesday, otherwise I'd say they can be cleaned up in the actual development of the tools

Reply:
@mexanick fixed